cms/templates/js/content-group-details.underscore

```diff
@@ -32,7 +32,7 @@
             </li>
         <% } else { %>
             <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
-                <button class="delete action-icon is-disabled" aria-disabled="true" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                <button class="delete action-icon is-disabled" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
```

Comment:
I think the one on top is a button that's clickable if you're allowed to delete, so the icon is only aria-hidden so screen readers don't read it out loud (but the button is still clickable). The one on the bottom (line 35) is always disabled because you're not allowed to delete.

Also, yes, it's definitely disabled="true". I think that was a typo on my part.